GameServerLib/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -536,6 +541,29 @@ public void UpdateSkin(int skinNo)
             Skin = skinNo;
         }
 
+        void IChampion.AddSpell(string name, byte slot, bool enabled)
```

Comment:
Maybe call this `SetSpell` instead, as it overrides any existing spell from said slot.